go/libkb/globals.go

```diff
@@ -83,6 +82,9 @@ type GlobalContext struct {
 	GregorDismisser     GregorDismisser        // for dismissing gregor items that we've handled
 	GregorListener      GregorListener         // for alerting about clients connecting and registering UI protocols
 	OutOfDateInfo       keybase1.OutOfDateInfo // Stores out of date messages we got from API server headers.
+
+	// Can be overloaded by tests to get an improvement in performance
+	NewTriplesec func(pw []byte, salt []byte) (Triplesec, error)
```

Comment:
A further protection would be to change this field to `newTriplesec`, add `SetNewTriplesec()` that tests can call that is very noisy, and a getter to get `newTriplesec`.